compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/ComposeScene.skiko.kt

```diff
@@ -450,6 +451,7 @@ class ComposeScene internal constructor(
      * @param button Represents the index of a button which state changed in this event. It's null
      * when there was no change of the buttons state or when button is not applicable (e.g. touch event).
      */
+    @OptIn(ExperimentalComposeUiApi::class)
```

Comment:
Is it considered as public API change?

Reply:
No. `OptIn` is compile-time annotation that supresses an error that we use experimental function inside.

It is fine to use experimental function inside, because they are implementation details, and are in the same module (if the experimental function in another module - we would have to support backward compatibility).